app/(app)/layout.tsx

```diff
@@ -78,14 +78,16 @@ export default async function RootLayout({
     : null;
 
   return (
-    <>
+    <div className="grid min-h-screen grid-rows-[auto_1fr_auto]">
```

Comment:
could you try a different approach instead of the `grid-rows-[auto_1fr_auto]`? Maybe using flex-col instead of grid. Thanks!